generator/templates/server/server.gotmpl

```diff
@@ -256,7 +256,6 @@ func (s *Server) SetAPI(api *{{ .Package }}.{{ pascalize .Name }}API) {
 	}
 
 	s.api = api
-	s.api.Logger = log.Printf
```

Comment:
You should regenerate bindata.go (this is done automatically if you enable .githooks on this repo).
Note that bindata is regenerated using go-bindata fork from Kevin's Burke

Reply:
done